packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,221 @@
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Usage](#usage)
+  - [Working with non-interactive triggers](#working-with-non-interactive-triggers)
+  - [Custom Triggers](#custom-triggers)
+- [Library](#library)
+  - [Bundle Size](#bundle-size)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop         | Type                                                                     | Default     | Description                                                                                                                             | Required |
+| ------------ | ------------------------------------------------------------------------ | ----------- | --------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| content      | `string`                                                                 | `undefined` | Content of the tooltip                                                                                                                  | ✅       |
+| children     | `React.ReactNode`                                                        | `undefined` | Trigger component for tooltip, Accepts any interactive element or icons                                                                 | ✅       |
+| placement    | `top, top-start, top-end, left, right, bottom, bottom-start, bottom-end` | `top`       | Placement of tooltip, the tooltip avoid collision with the edge of the screen and flip to the oppisite side even when placement is set. |          |
+| onOpenChange | `({ isOpen }) => void`                                                   | `undefined` | Called when tooltip isOpen state is changed, this can be used to detect when tooltip opens or closed                                    |          |
+
+### Examples:
+
+#### Usage
+
+```js
+// basic example
+<Tooltip content="Amount reversed to customer bank account" placement="top">
+  <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+</Tooltip>
+```
+
+### Working with non-interactive triggers
+
+To make tooltip apear on hovering over non-interactive elements such as icons, badges, counters etc. We will provide a `TooltipInteractiveWrapper` component which will work as a minimal trigger:
+
+```js
+import { Tooltip, TooltipInteractiveWrapper } from '@razorpay/blade';
+
+// non-interactive element as trigger
+<Tooltip content="Amount reversed to customer bank account">
+  <TooltipInteractiveWrapper>
```

Comment:
This might be a noob question but can we make it so that the `TooltipInteractiveWrapper`'s functionality is automatically added if the `Tooltip`'s children are not interactive?